info/contact.njk

```diff
@@ -8,9 +8,18 @@ eleventyNavigation:
 ---
 <section class="container contact">
     <h1>Contact</h1>
-    <p>Looking for support with the game Rust? <i>That's not us:</i> visit https://rust.facepunch.com/support</p>
-    <p>For all inquiries related to the Rust programming language, please email <a href="mailto:contact@rustfoundation.org">contact@rustfoundation.org</a>.</p>
+    
+    <div class="section">
+    <h2>We don't make the game Rust!</h2>
+    <p>Click or tap the logo for support with the game</p>
+    <a href="https://rust.facepunch.com/support"><img src="/img/rust-the-game.svg"></a>
```

Comment:
Do we have permission to use this logo?

Is it a good idea for us to have this logo on a Rust (language) website, or will it *add* to confusion?